docs/framework/deployment/windows/installing-dotnet-35-windows-10.md

```diff
@@ -0,0 +1,20 @@
+# Installation the .NET Framework 3.5 on Windows 10
```

Comment:
Actually the guidelines about ing is for filenames (https://opsdocs.azurewebsites.net/en-us/opsdocs/partnerdocs/SEO-checklist-for-migrating-content-to-ms-docs?branch=master)

But you can see that it ends up affecting title too: https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio